Examples/Swift/ViewController.swift

```diff
@@ -326,3 +289,16 @@ class CustomAnnotationView: MGLUserLocationAnnotationView {
         layer.borderColor = UIColor.white.cgColor
     }
 }
+
+class CustomStyle: DefaultStyle {
+    override func apply() {
+        super.apply()
+        ManeuverView.appearance().backgroundColor = #colorLiteral(red: 0.2974345386, green: 0.4338284135, blue: 0.9865127206, alpha: 1)
+        RouteTableViewHeaderView.appearance().backgroundColor = #colorLiteral(red: 0.2974345386, green: 0.4338284135, blue: 0.9865127206, alpha: 1)
+        
+        DistanceLabel.appearance().textColor = #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1)
+        DestinationLabel.appearance().textColor = #colorLiteral(red: 0.8039215803, green: 0.8039215803, blue: 0.8039215803, alpha: 1)
+        TimeRemainingLabel.appearance().textColor = #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1)
+        ArrivalTimeLabel.appearance().textColor = #colorLiteral(red: 0.8039215803, green: 0.8039215803, blue: 0.8039215803, alpha: 1)
```

Comment:
While we're at it, want to add a `StyleType` enum to this style and also add a custom dark theme? Would be cool to show custom night/day switching styles in the example.

Reply:
Fixed in https://github.com/mapbox/mapbox-navigation-ios/pull/536/commits/29b1269d2aa7c07ee75ea61598ca0c1c4d20b65f